deployment/k8s/charts/aggregates/migrationAssistant/values.yaml

```diff
@@ -9,17 +9,19 @@ conditionalPackageInstalls:
   replayer: true
   sharedConfigs: true
   snapshotVolume: true
+  logsVolume: true
```

Comment:
I suspect that we'll want to use otel-collector to lift and ship logs to a final resting place using compression.  This is a good quick-and-dirty fix that we can eventually make into false for prod and leave it as true for devs.